assemble/bin/accumulo-cluster

```diff
@@ -260,7 +260,7 @@ function start_here() {
   for group in $SSERVER_GROUPS; do
     for host in $local_hosts; do
       G="SSERVER_HOSTS_${group}"
-      for sserver in "${!G}"; do
+      for sserver in ${!G}; do
```

Comment:
I made this change because scan servers were not starting. I had two groups of three scan servers.  When I debugged it all three scan servers in a group would be consumed in one pass of the for loop.  When start_service got all three at once it silently did nothing (I think it expected the second argument to be sserver, compactor, etc but instead the 2nd arg was a hostname/ip).  Removing the quotes around `${!Q}` caused the for loop to go 3 times as expected.